lib/node_modules/@stdlib/math/base/special/binomcoeff/src/main.c

```diff
@@ -16,6 +16,8 @@
 * limitations under the License.
 */
 
+#include "stdlib/math/base/assert/is_integerf.h"
+#include "stdlib/math/base/assert/is_oddf.h"
 #include "stdlib/math/base/special/binomcoeff.h"
```

Comment:
```suggestion
#include "stdlib/math/base/special/binomcoeff.h"
#include "stdlib/math/base/assert/is_integerf.h"
#include "stdlib/math/base/assert/is_oddf.h"
```

Includes of dependencies should always come after the include of the package itself.